lib/local_system.rb

```diff
@@ -50,7 +50,7 @@ def validate_machinery_compatibility
         message = "Running Machinery is not supported on this system.\n" \
           "Supported operating systems are: #{supported_oses}"
 
-        raise(Machinery::Errors::IncompatibleHost.new(message))
+        Machinery::Ui.puts message
```

Comment:
I would prefer a more welcoming text here. Maybe something like "You are running Machinery on a platform we don't explicitly support and test. It still could work very well. If you run into issues or would like to provide us feedback, you are welcome to file an issue at https://github.com/SUSE/machinery/issues/new or write an email to machinery@lists.suse.com."

Any maybe it should be possible to switch this message off, or we should only show it once, because when you run Machinery regularly it might be very annoying.